### PR TITLE
Publishing our Windows dropgz version in linux dropgz manifest

### PR DESCRIPTION
**Pull request by rjdenney**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

File: .pipelines/pipeline.yaml [hunk 1]
```diff
@@ -281,7 +296,8 @@ stages:
           matrix:
             cni_dropgz:
               name: cni-dropgz
-              platforms: linux/amd64 linux/arm64
+              os_version: ltsc2019 ltsc2022
+              platforms: linux/amd64 linux/amd64 windows/amd64
             cni_dropgz_test:
               name: cni-dropgz-test
               platforms: linux/amd64 linux/arm64 windows/amd64
```
Comment (rbtr) [inline .pipelines/pipeline.yaml:303]: also needs `os_version`

File: .pipelines/pipeline.yaml [hunk 2]
```diff
@@ -281,7 +296,8 @@ stages:
           matrix:
             cni_dropgz:
               name: cni-dropgz
-              platforms: linux/amd64 linux/arm64
+              os_version: ltsc2019 ltsc2022
```
Comment (rbtr) [inline .pipelines/pipeline.yaml:299]: if adding `os_version`, Makefile needs to be updated to make the dropgz/dropgz-test manifest targets match [this](https://github.com/Azure/azure-container-networking/blob/4c761480c152eb39c5b710327ebfb8e50da7c4ac/Makefile#L559)

**Review by rbtr — APPROVED** [review]

**Review by rbtr — CHANGES_REQUESTED** [review]
meant to request changes 🙃

**Review by rbtr — APPROVED** [review]

**Review by pjohnst5 — APPROVED** [review]
Approved

**Review by pjohnst5 — APPROVED** [review]
LGTM

**Review by rbtr — APPROVED** [review]

**Pull request merged**